core/arch/arm/plat-hikey/platform_config.h

```diff
@@ -121,21 +121,33 @@
 
 #elif defined(PLATFORM_FLAVOR_hikey960)
 
+/*
+ * Physical address ranges for HiKey960 RAM:
+ * 3G board: 0~3G
+ * 4G board: 0~3G 3~3.5G 8~8.5G
+ * 4G board: 0~3G 3G~3.5G 8G~8.5G
```

Comment:
This particular line looks a bit confusing, since it's kind of "the same" with the previous line.

Reply:
Doh! Bad conflict resolution on my part as I used git `rebase -i`. Will fix.